### ISSUE TITLE
[Bug] Product detection issue, the Metrics ping doesn't include Fenix Beta in "metrics.mozilla_products"

### ISSUE THREAD

**Issue by AndiAJ** [mozilla-mobile/fenix#8206]
## Prerequisites
Have Fenix Beta installed on your device.

## Steps to reproduce
1. Generate a metrics ping

### Expected behavior
org.mozilla.fenix.beta **should be included** in the  "metrics.mozilla_products" from the metrics ping.

### Actual behavior
org.mozilla.fenix.beta is **not included** in the  "metrics.mozilla_products" from the metrics ping.

### Device information

* Android device: 
* Fenix version: Nightly #20380605 from 2/7

**Comment by mcarare** [comment]
@AndiAJ I can see `org.mozilla.firefox_beta` being added to `mozillaProducts` if Beta is installed. Can you please recheck this? Thank you!

**Comment by AndiAJ** [comment]
Hi **verified as fixed** on Firefox 85.0.0-beta.6 using a Google Pixel 3a (Android 11)

Ping 2a489317-697b-4036-844b-f87a0ae607c2
```
"metrics.mozilla_products": [
            "org.mozilla.firefox_beta"
          ]
```
[Logcat](https://github.com/mozilla-mobile/fenix/files/5786182/metrics.txt)
[Glean dashboard](https://debug-ping-preview.firebaseapp.com/pings/aj-t0801000)
